README.md

```diff
@@ -18,12 +18,18 @@ All the titles, images, descriptions, files, etc are described in [usecases.json
 * A [jenkins plan](https://bbpcode.epfl.ch/ci/job/nse.usecases-wizard.github/) is triggered.
 * The new changes appear in [Online Use Cases DEV](https://collab.humanbrainproject.eu/#/collab/8444/nav/64015)
 
-### To deploy in production: 
+### To deploy in production (non notebook): 
 * The owner receives the confirmation from the tests and the user that everything works correctly.
 * The owner create a pull request from `dev` environment to `master`.
 * A [jenkins plan](https://bbpcode.epfl.ch/ci/job/nse.usecases-wizard/) is triggered.
 * The new changes appear in [Brain Simulation Platform](https://collab.humanbrainproject.eu/#/collab/1655/nav/28538)
 
+### To deploy in production (notebook): 
+
+* The owner is not able to deploy directly a notebook in production.
+* If a notebook has been changed, updated or somehow modified, the owner of THIS repo must be notified. 
```

Comment:
This point should be after the last one (After testing....)
And I would suggest something like:
```
* If a notebook has been changed, updated or somehow modified, the owner of THIS repo must be notified.
* There is a file `prod_transform_notebooks.py` that needs to be executed in a Collab to update the content of the file keeping the production ID
* If the owners create a new notebook, this ID should be added in the `prod_transform_notebooks.py`
```